internal/commands/report_test.go

```diff
@@ -30,17 +39,19 @@ func testReportCommand(t *testing.T, when spec.G, it spec.S) {
 		packHome        string
 		packConfigPath  string
 		packMissingHome string
+		client          commands.PackClient
 	)
 
 	it.Before(func() {
 		var err error
 		logger = ilogging.NewLogWithWriters(&outBuf, &outBuf)
-		command = commands.Report(logger)
+		client = &fakeClient{}
+		command = commands.Report(logger, client)
 
 		packHome, err = ioutil.TempDir("", "")
 		h.AssertNil(t, err)
 		packConfigPath = filepath.Join(packHome, "config.toml")
-		h.AssertNil(t, ioutil.WriteFile(packConfigPath, []byte(`default-builder-image = "some/image"`), 0666))
+		h.AssertNil(t, ioutil.WriteFile(packConfigPath, []byte(`default-builder-image = "some/image"`), 0666)) // TODO we should probably restore the existing value of this
```

Comment:
This is a horrible bug. We should add an issue for this. I think we are trying to adhere to only adding TODOs in the code when there is a corresponding issue.

Reply:
Actually the bug I was thinking of is coming from set_default_builder_test.go. I just put up a [PR](#581) for this, it has been bothering me for awhile.

I believe this line of code is okay because `packConfigPath` is residing in a temporary directory.

Reply:
will clean up